lib/std/fs/test.zig

```diff
@@ -298,6 +298,25 @@ test "File.stat on a File that is a symlink returns Kind.sym_link" {
     }.impl);
 }
 
+test "Dir.statLink" {
```

Comment:
```suggestion
test "Dir.statLink" {
    // Either a wasmtime or wasi-libc bug: NO_FOLLOW flag is
    // dropped/ignored by fstatat when wasi-libc is used
    if (native_os == .wasi and builtin.link_libc) return error.SkipZigTest;

```

(see https://github.com/ziglang/zig/pull/20843#issuecomment-2261527722 for info)